client/client.js

```diff
@@ -0,0 +1,126 @@
+var log = require('minilog')('radar:client'),
+    Core = require('../core');
+
+function Client (name, id) {
+  this.createdAt = Date.now();
+  this.lastModified = Date.now();
+  this.name = name;
+  this.id = id;
+  this.subscriptions = {};
+  this.presences = {};
+};
+
+// 86400:  number of seconds in 1 day
+var DEFAULT_DATA_TTL = 86400;
+
+// Class properties
+Client.clients = {};            // keyed by name
+Client.names = {}               // keyed by id
+
+require('util').inherits(Client, require('events').EventEmitter);
+
+// Public API
+
+// Class methods
+
+// Set up client name/id association, client key, and return client
+Client.get = function(id, name, accountName) {
+  Client.names[id] = name;
+  var client = Client._get(id, name);
+  client.key = Client._keyGet(name, accountName);
+
+  return client;
+};
+
+// For a given client id/name pair, store the associated message
+Client.dataStore = function (id, message) {
```

Comment:
This can be a prototype method, so we can use this directly after we get the client, which I think is better.

```
var c = Client.get(id, name, account);

c.dataStore(message);
// similarly
c.dataLoad();
```